data/locale/ru-RU.ini

```diff
@@ -0,0 +1,38 @@
+NDIPlugin.Default="По умолчанию"
+NDIPlugin.NDISourceName="NDI™ источник"
+NDIPlugin.SourceProps.SourceName="Названия источника"
+NDIPlugin.SourceProps.Bandwidth="Пропускная способность"
+NDIPlugin.SourceProps.Sync="Синхронизация"
+NDIPlugin.SourceProps.HWAccel="Разрешить аппаратное ускорение"
+NDIPlugin.SourceProps.AlphaBlendingFix="Fix alpha blending (adds a filter to this source)"
```

Comment:
Please don't keep untranslated strings in your localization. These will fallback automatically to English if it can't be found in your Russian version.
```suggestion
```